llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,154 @@
+; REQUIRES: powerpc-registered-target
+; RUN: llc -filetype=obj -o %t -mtriple=powerpc-aix-ibm-xcoff < %s
+; RUN: llvm-symbolizer --obj=%t 'DATA 0x60' 'DATA 0x61' 'DATA 0x64' 'DATA 0X68' \
+; RUN:   'DATA 0x90' 'DATA 0x94' 'DATA 0X98' | \
+; RUN:   FileCheck %s
+
+;; This files tests llvm-symbolizer can successfully symbolize the data symbol
+;; from the DWARF info on AIX. Note that AIX is big endian.
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
+
+; CHECK: bss_global
+; CHECK-NEXT: 96 4
+; CHECK-NEXT: /t.cpp:1
+; CHECK-EMPTY:
```

Comment:
This pair of CHECKs needs two comments - one for each `bss_gloabl` instance, to highlight what is special about this particular case (i.e. the offsets, as you explained ebfore).